sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLPlannerImpl.java

```diff
@@ -211,7 +212,7 @@ public RelRoot rel(String sql, QueryParameters params) {
 
     RelNode convertedNode =
         QueryStatementConverter.convertRootQuery(context, (ResolvedQueryStmt) statement);
-
+    LOG.info("SQLPlan>\n" + RelOptUtil.toString(convertedNode));
```

Comment:
This should probably be `debug` level.

Reply:
oops. This is a line that was not added intentionally. Have removed it.